pygraphviz/agraph.py

```diff
@@ -1617,7 +1618,7 @@ def draw(self, path=None, format=None, prog=None, args=""):
         if path is not None:
             fh = self._get_fh(path, "w+b")
             fh.write(data)
-            if isinstance(path, str):
+            if isinstance(path, (str, pathlib.Path)):
```

Comment:
Maybe we should be testing this too?

Reply:
Currently https://github.com/pygraphviz/pygraphviz/blob/main/pygraphviz/tests/test_drawing.py is only passing Filehandles directly. Hence we would need no cover passing `str` as well as `pathlib.Path`.

Pytest should detect warnings automatically, but by default a `ResourceWarning` would not cause a test case to fail.

Reply:
I just added the test case in 5e82852664aa993f7d29eab58267782f98c1dd28, sorry for the delay!